token/client/src/client.rs

```diff
@@ -62,8 +62,14 @@ pub trait SendTransactionRpc: SendTransaction {
 #[derive(Debug, Clone, Copy, Default)]
 pub struct ProgramRpcClientSendTransaction;
```

Comment:
tbh the name `ProgramRpcClientSendTransaction` bothers me a lil bit now, since its used for rpc and offline, but i didnt think of anything better and the types here are already pretty complex

Reply:
I almost made the same comment, but I think it's technically ok since the base of both is meant to look like an RPC client, as opposed to a banks client.